internal/pkg/service/stream/storage/node/coordinator/filerotation/operator.go

```diff
@@ -372,67 +373,68 @@ func (o *operator) closeFile(ctx context.Context, file *fileData) {
 	ctx, cancel := context.WithTimeout(context.WithoutCancel(ctx), o.config.FileCloseTimeout.Duration())
 	defer cancel()
 
-	o.logger.Info(ctx, "closing file")
+	// Wait for all slices upload, get statistics
+	stats, opErr := o.waitForFileClosing(ctx, file)
 
-	// Wait until all file slices are uploaded
-	if err := o.waitForSlicesUpload(ctx, file.FileKey); err != nil {
-		err = errors.PrefixError(err, "error when waiting for file slices upload")
-		o.logger.Error(ctx, err.Error())
-		return
-	}
```

Comment:
There was some stuck slice upload (another/fixed bug), and the file import was therefore blocked.

But error from `waitForSlicesUpload` was not increasing retry attempt, so it was spamming logs every second.